### PR TITLE
Removed duplicate file and added full path for demo-manifest In Readme

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/msgext-search/python/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ build a Search-based Messaging Extension.
 ## Try it yourself - experience the App in your Microsoft Teams client
 Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
 
-**Teams Messaging Extensions Search:** [Manifest](/samples/msgext-search/csharp/demo-manifest/msgext-search.zip)
+**Teams Messaging Extensions Search:** [Manifest](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msgext-search/csharp/demo-manifest)
```
Comment (Pawank-MSFT) [inline samples/msgext-search/python/README.md]: Add "/msgext-search.zip" also

**Review by Pawank-MSFT — COMMENTED** [review]
Please fix

File: samples/msgext-search/python/README.md (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/msgext-search/python/README.md]: fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**